go.mod

```diff
@@ -1,11 +1,21 @@
 module github.com/pbs/gorson
 
-go 1.15
+go 1.16
 
 require (
```

Comment:
I am on the fence about this file... on one hand it harks back to the days of `pip freeze > requirements.txt` days that put all the dependencies of dependencies into the `requirements.txt` file. And it seems unintuitive with `go.sum` being a thing. Should we have all these ``// indirect` packages listed here?

Reply:
Yeah, good question. Beyond "use go modules", I don't know what the state of the art is in go dependency management. An opportunity for research, if you're up for it :)

Otherwise, I think this is ok.

Reply:
I asked the question in the DCTech slack channel, see what other people think... weirdly enough, we actually need one of those `indirect` packages to be listed here for the project to build. I am leaning on just keeping them all because of that fact.